app/cdash/app/Model/Build.php

```diff
@@ -2116,8 +2116,8 @@ public function UpdateParentTestNumbers($newFailed, $newNotRun, $newPassed)
                     'testpassed' => $numPassed,
                 ]);
 
-        // NOTE: as far as I can tell, build.testtimestatusfailed isn't used,
-        // so for now it isn't being updated for parent builds.
+            // NOTE: as far as I can tell, build.testtimestatusfailed isn't used,
+            // so for now it isn't being updated for parent builds.
```

Comment:
The linter made this change, but it doesn't seem to be directly related to any code changes I made.  Furthermore, I am unable to reproduce any linting failures locally, despite the CI failing [here](https://app.circleci.com/pipelines/github/Kitware/CDash/1015/workflows/caab9e83-be1e-4a69-aea8-36884845e3a2/jobs/4515?invite=true#step-104-348), with further changes to this file suggested.

Yet another weird issue with the style check, in a continuation of the issues described in https://github.com/Kitware/CDash/issues/1309.

Reply:
Issue seems to have been resolved by reverting the changes to `Build.php`.  I am possibly running a different version of the style checker.  This type of issue would be fixed by pinning all of our dependencies, as discussed in https://github.com/Kitware/CDash/issues/1313.